BSD: add docs for OPNSense and pfSense (#996)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/network/pfsense.md

name: pfSense
type: intake

## Overview

pfSense is a firewall software distribution based on FreeBSD.

This intake ingest filterlog from pfSense.

!!! warning
    This format is still in beta, please use it wisely.

{!_shared_content/operations_center/detection/generated/suggested_rules_8510051d-c7cf-4b0c-a398-031afe91faa0_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/8510051d-c7cf-4b0c-a398-031afe91faa0.md!}

## Configure

This setup guide will show you how to forward your Firewall logs
to SEKOIA.IO by means of a syslog transport channel.

### Prerequisites

- Have an internal log concentrator

### Enable Syslog forwarding

> Following the [official guide](https://docs.netgate.com/pfsense/en/latest/monitoring/logs/remote.html) to enable syslog forwarding from your pfSense firewall to the internal log concentrator.

Reviewer: ```suggestion
1. Follow the [official guide](https://docs.netgate.com/pfsense/en/latest/monitoring/logs/remote.html) to enable syslog forwarding from your pfSense firewall to the internal log concentrator.
```

> Define your log concentrator as a remote syslog server and select `Firewall Events` as the remote syslog content.

Reviewer: ```suggestion
2. Define your log concentrator as a remote syslog server and select `Firewall Events` as the remote syslog content.
```

> Save your changes.

Reviewer: ```suggestion

3. Save your changes.
```
